flexmeasures/api/v3_0/assets.py

```diff
@@ -152,13 +152,28 @@ def index(
         if page is not None:
             if per_page is None:
                 per_page = 10
-            assets = db.paginate(query, per_page=per_page, page=page).items
+            from flask_sqlalchemy.pagination import SelectPagination
 
-            # TODO: return total number of records. e.g. {"data" : [], "num-records" : 120, "filtered-records" : 10}
+            select_pagination: SelectPagination = db.paginate(
+                query, per_page=per_page, page=page
+            )
+
+            assets = {
+                "data": select_pagination.items,
+                "num-records": select_pagination.total,
```

Comment:
According to [the DataTables documentation](https://datatables.net/manual/server-side#Returned-data), the `num-records` parameter should be:

```
Total records, before filtering (i.e. the total number of records in the database)
```

Either we implement that (with a `count` on the table, or using your new util function `get_num_accessible_assets()`) or we add a comment here that we are not implementing it that way.